.golangci.yml

```diff
@@ -24,6 +24,10 @@ issues:
       linters:
         - golint
       text: "exported var Translator should have its own declaration"
+    - path: hash/structhash/structhash_test.go
+      linters:
+        - maligned
+      text: "struct of size 136 bytes could be of size 128 bytes"
```

Comment:
Specific linter exclusion from a specific file. This is done in the config to not put ugly `//nolint:` all over the code. Also, you have more control on excluded linter, because you can pick only one thing from linter to exclude, opposti to `//nolint:` directive where you exclude entire linter.